Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM openshift/origin-release:golang-1.10 as builder
+WORKDIR /go/src/github.com/operator-framework/operator-marketplace
+COPY . .
+RUN make build
+
+FROM centos:7
+RUN useradd marketplace-operator
+USER marketplace-operator
+COPY --from=builder /go/src/github.com/operator-framework/operator-marketplace/tmp/_output/bin/marketplace-operator /usr/bin
+
+LABEL io.k8s.display-name="OpenShift marketplace-operator" \
+      io.k8s.description="This is a component of OpenShift Container Platform and manages the OpenShift Marketplace." \
+      io.openshift.tags="openshift" \
```

Comment:
I would recommend `io.openshift.tags=openshift,marketplace` to be more specific.

Reply:
We should follow whatever convention that needs to be followed for these tags.